Source/LibationWinForms/Form1.ProcessQueue.cs

```diff
@@ -23,17 +23,7 @@ private void Configure_ProcessQueue()
 
 		private void ProductsGrid_LiberateClicked(object sender, LibraryBook e)
 		{
```

Comment:
Per email conversation: I think nothing is needed here. I dunno what the hell went wrong this morning

Reply:
Right. Reverted.